BO4E/ENUM/MengenZusatzInformation.cs

```diff
@@ -0,0 +1,32 @@
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// SG10 QTY 6063
+    /// </summary>
+    public enum MengenZusatzInformation
+    {
+        /// <summary> Wahrer Wert: 220 </summary>
+        WAHRER_WERT,
+
+        /// <summary> Ersatzwert: 67 </summary>
+        ERSATZWERT,
+
+        /// <summary> Vorläufiger Wert: Z18 </summary>
+        VORLAUUFIGER_WERT,
+
+        /// <summary> Angabe für Lieferschein: Z31 </summary>
+        ANGABE_FUER_LIEFERSCHEIN,
+
+        /// <summary> Vorschlagswert: 201 </summary>
+        VORSCHLAGSWERT,
+
+        /// <summary> Nicht verwendbarer Wert: 20 </summary>
+        NICHT_VERWENDBARER_WERT,
+
+        /// <summary> Prognosewert: 187 </summary>
+        PROGNOSEWERT,
+
+        /// <summary> Energiemenge summiert: 79 </summary>
+        ENERGIEMENGE_SUMMIERT,
```

Comment:
dieses mapping auf edifact machen wir (in einem mittlerweile deprecateten ansatz) noch anders. mit einem edienum-attribute. aber weil deprecated können wir das auch so lassen wie es ist.

Reply:
Was wäre denn die Moderne variante?

Reply:
edifact-bo4e-converter ;)

Reply:
Ich benötige aber einen Platz um diese Inforamtion zu übertragen. Was schlägst du vor?

Reply:
remarks?

Reply:
Okay. Wir müssen reden.